data/src/main/java/com/github/abdallahabdelfattah13/data/repositories/mapper/MapperData.kt

```diff
@@ -0,0 +1,19 @@
+package com.github.abdallahabdelfattah13.data.repositories.mapper
+
+import com.github.abdallahabdelfattah13.data.repositories.entities.FeedEntity
+import com.github.abdallahabdelfattah13.domain.model.Feed
+
+class MapperData {
+
+    companion object {
+
+        fun toFeedDomain(feedEntity: List<FeedEntity>): List<Feed> {
+            return feedEntity
+                .map { Feed(it.id, it.url, it.thumbnail, it.title, it.description) }
+        }
+
+        fun toFeedData(feed: Feed): FeedEntity = FeedEntity(feed.id, feed.url, feed.thumbnail, feed.title, feed.description)
+
+    }
```

Comment:
It's not a good choice use companion object for this. You can solve this problem in two ways.

1. Create an extension function on FeedEntity like `fun FeedEntity.mapToFeed(): Feed`
2. Remove those methods from the Companion Object and instantiate a classe to do this.

The reason it's not a good option is that:

1. It increases the dex size unnecessary
2. This is not a kotlin way of code.

Reply:
Yes, it's better to use extension function. Thanks